src/syngen/ml/vae/models/dataset.py

```diff
@@ -75,11 +76,26 @@ def _predefine_fields(self):
         self.uuid_columns = set()
         self.uuid_columns_types = {}
         self.dropped_columns = fetch_training_config(self.paths["train_config_pickle_path"]).dropped_columns
+        self.non_existent_columns = set()
         self.order_of_columns = self.df.columns.tolist()
 
     def __prepare_dir(self):
         os.makedirs(self.paths["fk_kde_path"], exist_ok=True)
 
+    def _check_for_non_existent_columns(self, keys_mapping: Dict, key_type: str):
+        """
+        Check if there are columns in the dataframe that are not in the metadata
```

Comment:
I think, it should be "Check if there are columns in metadata that are not present in the dataframe."